mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java

```diff
@@ -0,0 +1,115 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.mediapipe.apps.instantmotiontracking;
+
+import java.util.ArrayList;
+
+// Stickers represent a unique object to render and manipulate in an AR scene.
+// A sticker has a sticker_id (a unique integer identifying a sticker object to
+// render), x and y normalized anchor coordinates [0.0-1.0], user inputs for
+// rotation in radians, scaling, and a renderID (another unique integer which
+// determines what object model to render for this unique sticker).
+
+public class Sticker {
+
+  private int renderID = 0; // ID of object to render
+  private float xAnchor, yAnchor; // Normalized X and Y coordinates of anchor
+  private float userRotation, userScaling; // User alterations to scaling and rotation
+  private int stickerID; // Unique sticker integer ID
+  private static int globalIDLimit = 1; // Used to determine next stickerID
+
+  public Sticker() {
+    this.renderID = 0;
+    stickerID = (Sticker.globalIDLimit++);
+  }
+
+  /** Sets a new asset rendering ID as defined by Mediapipe graph. * */
+  public Sticker(int renderID) {
+    this.renderID = renderID;
+    stickerID = (Sticker.globalIDLimit++);
+  }
+
+  /** Return sticker ID integer. * */
+  public int getStickerID() {
+    return this.stickerID;
+  }
+
+  /** Sets new normalized X and Y anchor coordinates. * */
+  public void setNewAnchor(float normalizedX, float normalizedY) {
+    this.xAnchor = normalizedX;
+    this.yAnchor = normalizedY;
+  }
+
+  /** Return integer array of format int[]{xAnchor,yAnchor}. * */
+  public float[] getAnchor() {
+    return new float[] {xAnchor, yAnchor};
+  }
+
+  /** Returns render asset ID used by Mediapipe graph. * */
+  public int getRenderAssetID() {
+    return renderID;
+  }
+
+  public void setRenderAssetID(int newRenderID) {
+    this.renderID = newRenderID;
+  }
+
+  /** Sets new user input of y-axis rotation (objective rotation, does not increment). * */
+  public void setRotation(float radians) {
+    this.userRotation = radians;
+  }
+
+  /** Return radians of rotation on the object set by user. * */
+  public float getRotation() {
+    return this.userRotation;
+  }
+
+  /** Sets pixel-length of new scalling (objective scaling, does not increment). * */
+  public void setScaling(float scaling) {
+    this.userScaling = scaling;
+  }
+
+  public float getScaling() {
+    return this.userScaling;
+  }
+
+  /**
+   * This method converts an ArrayList of stickers to a data string that can be passed to the
+   * Mediapipe graph.
+   *
+   * @param stickerList ArrayList of Sticker objects to convert to data string
+   * @return String of all sticker data (Managed by Android end)
+   */
+  public static String stickerArrayListToRawData(ArrayList<Sticker> stickerList) {
```

Comment:
Let's add a TODO to use something more structured/efficient for this in the future, like a proto.